docker/docker-compose.yml

```diff
@@ -186,13 +194,16 @@ services:
   workflow:
     healthcheck:
       test: [ "CMD", "curl", "-f", "http://localhost:8080/actuator/health" ]
-      start_period: 240s
+      start_period: 3m
+      timeout: 30s
+      retries: 3
     image: quay.io/ukhomeofficedigital/hocs-workflow:${WORKFLOW_TAG:-branch-main}
     ports:
       - 8091:8080
     networks:
       - hocs-network
     environment:
+      JAVA_OPTS: '-Xms1536M -Xmx2G'
```

Comment:
huh. is java not clever enough to know how much RAM to use? Do we need to do this for the other services too?